### ISSUE TITLE
Support zoom control properties on WebView

### ISSUE THREAD

**Issue by jonek** [eclipsesource/tabris-js#255]
We should support the platform's native features to customize zooming behavior on WebView.

For example on Android the following two properties control if the user is able to use the pinch-to-zoom gesture on a web view.
```java
settings = webView.getSettings();
settings.setBuiltInZoomControls( true );
settings.setDisplayZoomControls( false );
```

We could also just always enable this settings per default. The user is then able to control the zooming behavior by add the following tag to a html page's header:
```html
<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=no">
```

By switching the `user-scalable` attribute between `yes` and `no`, the user would be able to alter zooming behavior from within the displayed html resource. Of course this is only possible iff the user has control over the html resource.

**Comment by jonek** [comment]
Currently Android behaves like `setBuiltInZoomControls( false )`. Thus the user currently is unable to zoom at all.

**Comment by cpetrov** [comment]
In the iOS implementation the user can zoom by default.

**Comment by mpost** [comment]
We will change the default behavior of the WebView on Android to support zooming and be zoomed out by default.

**Comment by mpost** [comment]
Implemented for the upcoming 1.7 release.
